### ISSUE TITLE
Missing LICENSE in wheels

### ISSUE THREAD

**Issue by SeaHOH** [py2exe/py2exe#184]
The released wheels did not included neither `MPL2-License.txt` nor `MIT-License.txt`.

https://peps.python.org/pep-0639/#add-license-expression-field
https://peps.python.org/pep-0639/#add-license-file-field
https://peps.python.org/pep-0639/#appendix-examples

Also, GitHub detected `MPL2-License.txt` only, but not `MIT-License.txt`.

**Comment by albertosottile** [comment]
The licenses are correctly mentioned on PyPI https://pypi.org/project/py2exe/ but, it might be true that the actual .txt files are not embedded in wheels (and legally I guess they should, though IANAL...). I will patch the setup scripts to do so. 

Regarding what GitHub detected, I am not sure how to fix that...

**Comment by albertosottile** [comment]
@SeaHOH Actually, I double checked: the LICENSE.txt file is in the wheels in the dist info (Python only accepts one license file there), whereas the other txt licenses are in the `py2exe` package itself. Therefore, the wheels contain all the license text files, so I would close this issue.

**Comment by SeaHOH** [comment]
Set License like this:
```
        license="MIT/X11 OR (MPL 2.0)",
        license_files=[
            "LICENSE.txt",
            "MIT-License.txt",
            "MPL2-License.txt",
        ],
```
![l](https://github.com/py2exe/py2exe/assets/8470820/66fc897f-6f9d-49a0-8267-6b7a038a5527)

**Comment by albertosottile** [comment]
@SeaHOH I will implement this in the next major. Thanks.

**Comment by albertosottile** [comment]
Fixed in 0.13.0.2
